### DISCUSSION TITLE
How to tell dependebot ignore a file?

### DISCUSSION THREAD

**Discussion by ssabzevari-antuit** [community/community#57963] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We are using Databricks and created a minimal db10_4_requirement.txt that corresponds to specific databricks runtime so our team can set their venv on their laptop. Dependbot keeps giving me upgrade suggestions which I am not interested in. How can I tell it to ignore this file?

**Comment by Gustavonn07** [comment]
I did a little research and did this step by step:
Here's an example of how you can configure Dependabot to ignore the db10_4_requirement.txt file:

1. Create a .dependabot directory in the root of your repository if it doesn't already exist.

2. Inside the .dependabot directory, create a config.yml file.

3. Add the following content to the config.yml file:

![image](https://github.com/community/community/assets/84361085/98374f69-e114-44d2-be8c-b02b824e6ee2)

 
This configuration specifies that Dependabot should ignore updates for the db10_4_requirement.txt file located in the root directory ("/") of the repository.

4. Commit and push the .dependabot/config.yml file to your repository.


I hope this solves your problem!

**Reply by ssabzevari-antuit** [reply]
Thanks I will test and let you know. Care to share the links?

**Reply by Gustavonn07** [reply]
I used these links:
1. https://jtemporal.com/requirements-txt/
2. https://www.databricks.com/resources/ebook/the-data-lakehouse-platform-for-dummies?utm_medium=paid+search&utm_source=google&utm_campaign=19774681654&utm_adgroup=146097664065&utm_content=ebook&utm_offer=the-data-lakehouse-platform-for-dummies&utm_ad=650353890392&utm_term=databricks&gclid=CjwKCAjw-b-kBhB-EiwA4fvKrIgz_NzPXTe6oXDhE9rKklnY2bSsUVpq40lllIWHp74x4Ybi2qvu5xoCMgoQAvD_BwE
3. https://chat.openai.com/
4. https://www.alura.com.br/artigos/databricks-o-que-e-para-que-serve?gclid=CjwKCAjw-b-kBhB-EiwA4fvKrCGcuNTYvcVuC-KKgiAAIOVo60XFbLgrB2K9izWkC0ki_C6er6P-cxoCW5EQAvD_BwE


Maybe there's a problem, since I'm Brazilian and I speak Portuguese so the links I used are in Portuguese (I don't know if they have a good translation)
